release_autogen_client/update_auto_gen_client_specific_version.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Usage: $ ./update_semantic_version_autogen_client.sh [ major | minor | patch ]
+# This script will update the semantic versioning for a client
+# based on the type specified. Default is 'patch'
+set -e
+set +x
+
+source ./common_packaging_utils.sh
+
+export new_version=${1}
+echo "new_version is ${1}"
+if [ -z "${new_version}" ]; then
+  echo "required parameter 'version' not provided. Please specify in form {major}.{minor}.{patch}"
+  exit 1
+fi
+
+setup_repos
+refresh_codegen
```

Comment:
Couldn't all of these scripts be simplified down to one script which does:

check out marquez-python-autogen in a tmp dir
set the version in marquez-python-autogen's config.json
generate the client
commit to marquez-python-autogen
push to marquez-python-autogen to pypi?

Reply:
Sure, happy to do that. The only exception is the last step, which I feel is better handled by CircleCI, in part b/c it very conveniently keeps the credentials safe, but also b/c I feel more comfortable with CircleCI doing the push to PyPi since it will keep a record about exactly what happened. Does that seem okay?

Reply:
Yep, that's fine.